frontend/src/pages/Navbar/SideNavCard.js

```diff
@@ -122,15 +122,16 @@ const SideNavCard = ({
       </List>
     ) : null}
     <List>
-      <Subheader> {strings.users.groups} </Subheader>
-      {groups.map(group => (
+      {groups.length ? <Subheader> {strings.users.groups} </Subheader> : null}
+      {groups.map(group => {
```

Comment:
fyi: you don't need a return statement here, just the parentheses are enough

Reply:
I tried it, but it didn't work without return